docs/0.3/using_purchase_order.md

```diff
@@ -230,102 +214,70 @@ public key file in the next command.
     gnrl $(cat ~/.grid/keys/beta-agent.pub) \
     --active \
     --role admin \
-    --role po-manager`
+    --role po-manager
     ```
 
 ### Create Purchase Orders
 
 {:start=1}
 
 1.  Using the `gridd-beta` command line, propose a draft purchase order on behalf
-    of Fred's Farm.
+    of MyOrg.
 
     ```
     root@gridd-beta:/# grid po create \
-    --org farm \
-    --id po_number: 0123456 \
-    --workflow-status Issued \
+    --buyer-org gnrl \
+    --seller-org myorg \
+    --alternate-id po_number:0123456 \
+    --workflow-state create \
+    --workflow-id built-in::collaborative::v1 \
     --wait 100
 
     root@gridd-beta:/# grid po version create \
-    --org farm \
-    --po 0123456 \
+    po_number:0123456 \
+    1 \
     --order-xml <path to order> \
```

Comment:
We need to provide an example purchase order version doc that users can use here